k8s/demo/common/ccd/ccd-full.yaml

```diff
@@ -6,7 +6,19 @@ metadata:
   namespace: ccd
   annotations:
     flux.weave.works/automated: 'true'
-    flux.weave.works/tag.java: 'glob:prod-*'
+    flux.weave.works/tag.ccd-full-admin-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-api-gateway: glob:prod-*
+    flux.weave.works/tag.ccd-full-case-management-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-data-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-definition-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-dm-store: glob:prod-*
```

Comment:
auto update of all dependencies required or just ccd components enough?

Reply:
better update al components